lib/datadog/tracing/contrib/utils/quantization/http.rb

```diff
@@ -120,8 +120,25 @@ def obfuscate_query(query, options = {})
               options[:regex] = nil if options[:regex] == :internal
               re = options[:regex] || OBFUSCATOR_REGEX
               with = options[:with] || OBFUSCATOR_WITH
+              keys = options.fetch(:keys, true)
```

Comment:
Is this a new custom option we are adding to our Ruby obfuscator, so something that's well defined inside Datadog already?